queries.py

```diff
@@ -87,3 +88,52 @@ def get_expenditure_by_country_func_econ_year():
     """
     df = execute_query(query)
     return df
+
+
+def get_basic_country_data(countries):
+    country_list = "', '".join(countries)
+    query = """
+        SELECT * FROM indicator.country
```

Comment:
do we need all the columns? 
I noticed that the browser is much slower on this branch. (but not affected in incognito mode)
We might be caching unnecessary data? Let's try and save some space.